lib/Bio/EnsEMBL/Versioning/Pipeline/PipeConfig/Source_update_conf.pm

```diff
@@ -112,36 +113,34 @@ sub pipeline_analyses {
       {
         -logic_name => 'ParseSource',
         -module => 'Bio::EnsEMBL::Versioning::Pipeline::ParseSource',
-        -parameters => {
-
-        },
         -max_retry_count => 0, # low to prevent pointless parsing repetition until someone can get attend to the problem.
         -hive_capacity => 20,
         -failed_job_tolerance => 25, # percent of jobs that can fail while allowing the pipeline to complete.
         -rc_name => 'mem',
         -flow_into => {
-          2 => ['?accu_name=error_bucket&accu_address={source}[]']
+          2 => ['?accu_name=error_bucket&accu_address=[]']
         }
       },
       {
         -logic_name => 'CollateIndexes',
         -module => 'Bio::EnsEMBL::Versioning::Pipeline::CollateIndexes',
         -max_retry_count => 0,
         -hive_capacity => 100,
-        -rc_name => 'normal'
+        -rc_name => 'normal',
+        -flow_into => {
+          2 => ['?accu_name=error_bucket&accu_address={source_name}']  
```

Comment:
error_bucket is being dataflown out as an arrayref by CollateIndexes, so it will appear as an arrayref to consumers of this accu (e.g. each value of the hashref $error_bucket will be an arrayref)